packages/nimble-components/src/wafer-map/tests/wafer-map.stories.ts

```diff
@@ -96,21 +96,24 @@ const metadata: Meta<WaferMapArgs> = {
             WARNING - The wafermap is still in development and considered
             experimental. It is not recommended for application use.
```

Comment:
The rendering for matrix.stories seems to be broken, with the wafer being very large and no dies being displayed. (You can see this in the [Chromatic diffs](https://www.chromatic.com/test?appId=60e89457a987cf003efc0a5b&id=639c8075fa885e71d319ad09))

Reply:
Yes ... this is strange indeed. We will have to revisit these tests.

Reply:
Unfortunately I will not have time today ... I took a quick look at the existing tests - could it be because no values are being passed to the "viewTemplates" used by the "createMatrix" functions ? Like here : 
![image](https://user-images.githubusercontent.com/35329597/208142754-6e5d205e-45af-4165-a46b-55c14c7ac80e.png)  ... if this is the case, how come it is working in the "main" ? The matrix stories were not modified ...

Reply:
I did a bit of debugging and it looks like `canvasSideLength` is still set to 0 in the `render()` function so it returns early. In the non-matrix story it has a value of 500. I wasn't able to immediately trace how it got set in the non-matrix story and why that didn't happen in the matrix story.

Reply:
I believe I did find the issue - fixed.